examples/gno.land/p/gov/dao/proposals.gno

```diff
@@ -0,0 +1,51 @@
+package dao
+
+import "std"
+
+type ProposalStatus string
+
+// ACTIVE -> ACCEPTED -> EXECUTION(SUCCEEDED/FAILED)
+// ACTIVE -> NOT ACCEPTED
```

Comment:
Expired?

Reply:
I think the approach we're rolling with is that the proposal can't be expired, but the executor can be